### PR TITLE
Add logging of elapsed time for the action.

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.Core/Infrastructure/MvcRouteHandler.cs [hunk 1]
```diff
@@ -94,10 +94,14 @@ public VirtualPathData GetVirtualPath(VirtualPathContext context)
...
                 {
-                    _logger.LogVerbose("Executing action {ActionDisplayName}", actionDescriptor.DisplayName);
+                    _logger.ExecutingAction(actionDescriptor);
+
+                    var startTime = Environment.TickCount;
 
                     await InvokeActionAsync(context, actionDescriptor);
                     context.IsHandled = true;
+
+                    _logger.ExecutedAction(actionDescriptor, Environment.TickCount - startTime);
```
Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Core/Infrastructure/MvcRouteHandler.cs]: This isn't currently correct by the way, ticks aren't milliseconds. Derp.

File: test/Microsoft.AspNet.Mvc.Core.Test/Infrastructure/MvcRouteHandlerTests.cs [hunk 2]
```diff
@@ -59,7 +65,7 @@ public class MvcRouteHandlerTests
                 loggerFactory: loggerFactory);
 
             var handler = new MvcRouteHandler();
-            var expectedMessage = "No actions matched the current request.";
```
Comment (kichalla) [inline test/Microsoft.AspNet.Mvc.Core.Test/Infrastructure/MvcRouteHandlerTests.cs]: keep the `.` as it is

File: src/Microsoft.AspNet.Mvc.Core/Logging/MvcRouteHandlerLoggerExtensions.cs [hunk 3]
```diff
@@ -10,11 +10,43 @@ namespace Microsoft.AspNet.Mvc.Logging
...
+
+        static MvcRouteHandlerLoggerExtensions()
+        {
+            _actionExecuting = LoggerMessage.Define<string>(
+                LogLevel.Verbose,
+                1,
+                "Executing action {ActionName}");
+
+            _actionExecuted = LoggerMessage.Define<string, double>(
+                LogLevel.Information,
```
Comment (kichalla) [inline src/Microsoft.AspNet.Mvc.Core/Logging/MvcRouteHandlerLoggerExtensions.cs]: curious...is this intentional...verbose or info?

File: test/Microsoft.AspNet.Mvc.Core.Test/Infrastructure/MvcRouteHandlerTests.cs (hunk 2 above)
Comment (rynowak) [inline test/Microsoft.AspNet.Mvc.Core.Test/Infrastructure/MvcRouteHandlerTests.cs]: No, none of our other stuff has periods.

File: src/Microsoft.AspNet.Mvc.Core/Logging/MvcRouteHandlerLoggerExtensions.cs (hunk 3 above)
Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Core/Logging/MvcRouteHandlerLoggerExtensions.cs]: Intentionally verbose.

**Comment by kichalla** [comment]
:shipit:

**Pull request merged**